string name
---
int32 id
float32 timeout
bool task_scheduling
bool localisation
bool path_planing
